sklearn/metrics/pairwise.py

```diff
@@ -1284,6 +1284,27 @@ def pairwise_distances_chunked(X, Y=None, reduce_func=None,
         yield D_chunk
 
 
+def set_diagonal_pairwise(distances, X, Y):
```

Comment:
This should be private, i.e. prefixed by _

Reply:
done